src/Middleware/PageDecorationMiddleware.php

```diff
@@ -35,26 +35,35 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
         $loris   = $request->getAttribute("loris");
         $page    = $request->getAttribute("pageclass")
         ?? new \NDB_Page($loris, new \NullModule($loris), "", "", "", "");
+
+        $DB            = \NDB_Factory::singleton()->database();
+        $language_rows = $DB->pselect("SELECT language_code, language_label FROM language", []);
```

Comment:
This query should probably not be here but I have no idea where to place it.

Reply:
Here looks reasonable to me but can't this loop be avoided (or at least hidden) by just using `pselectColWithIndexKey`?